echo/server/.env.sample

```
@@ -1,94 +1,94 @@
+LITELLM_WHISPER_URL=https://api.openai.com/v1
+
+RUNPOD_WHISPER_MAX_REQUEST_THRESHOLD=30
+DISABLE_CHAT_TITLE_GENERATION=1
+
+############################################################
+# Flags / Debug
+############################################################
+DEBUG_MODE=0
+DISABLE_SENTRY=0
+SERVE_API_DOCS=0
 DISABLE_REDACTION=1
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)